core/http-auth-aws/src/main/java/software/amazon/awssdk/http/auth/aws/AwsV4aHttpSigner.java

```diff
@@ -31,15 +32,15 @@
 @SdkPublicApi
 public interface AwsV4aHttpSigner extends AwsV4FamilyHttpSigner, HttpSigner<AwsCredentialsIdentity> {
     /**
-     * The AWS region name to be used for computing the signature. This property is required.
-     * TODO(sra-identity-and-auth): Should this be a list or rename to SIGNING_SCOPE?
+     * The AWS region-set to be used for computing the signature. This property is required.
```

Comment:
Do we want to have a default? Global (i.e. `*`)?

Reply:
Looks like having it required works for our use from endpoints based sigv4a. I don't feel strongly about either option. Starting required is a 2-way door and we can make '*' default later too.